ert3/config/_stages_config.py

```diff
@@ -35,20 +52,40 @@ def is_executable(cls, location):
 
 class Step(_StagesConfig):
     name: str
-    script: List[str]
+    type: str
```

Comment:
For now yes there are only two options, but I don't expect a lot more in the near future so a `Literal` is a good suggestion. Thanks.

Reply:
It seems that:
https://pydantic-docs.helpmanual.io/usage/types/#literal-type 
Literal Type is a new feature of the python standard library as of python 3.8; prior to python 3.8, it requires the typing-extensions package.

So it does not really work for py36 without typing-extensions (https://pypi.org/project/typing-extensions/) I would like to avoid adding it as a dependency with this PR

Reply:
It is already in use in the experiment config. So you will not have to do more than include it :)